core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/request/StackScaleV4Request.java

```diff
@@ -97,4 +97,12 @@ public void setThreshold(Long threshold) {
     public Long getThreshold() {
         return threshold;
     }
+
+    public NetworkScaleV4Request getNetworkScaleV4Request() {
```

Comment:
getNetworkScaleV4Request and setNetworkScaleV4Request are not used. I removed it and did not observe any build issues.